lib/json_api_controller/precondition_check.rb

```diff
@@ -0,0 +1,28 @@
+module JsonApiController
+  module PreconditionCheck
+    def precondition_check
+      if !precondition
+        raise PreconditionNotPresent, precondition_error_msg
+      elsif precondition_fails?
+        raise PreconditionFailed
+      end
+    end
+    
+    def precondition
+      case action_name
+      when "show"
```

Comment:
We could. I think it'd be a little hostile for API exploration, which is why I didn't require them. I can also get rid of this condition since its not being used.